hail_batch_dev.md

```diff
@@ -152,3 +152,33 @@ When you need to debug an issue within your namespace, it's often helpful to ins
 kubectl --namespace $NAMESPACE get pod
 kubectl --namespace $NAMESPACE logs $POD
 ```
+
+## Merging upstream changes
+
+We try to keep our Hail fork as close as possible to the upstream repository. About once a week we integrate any upstream changes as follows:
+
+```bash
+git remote add upstream https://github.com/hail-is/hail.git  # One-time setup.
+
+git fetch origin
+git fetch upstream
+git checkout -b upstream
+git reset --hard origin/main
+git merge upstream/main  # Potentially resolve any conflicts.
+git push origin upstream  # Create a PR as usual.
+```
+
+## Deploying changes to production
+
+After a change has been merged to the `main` branch, it can be deployed to the `default` namespace using the `prod_deploy` API endpoint. This will always use the current `HEAD`. Similar to a `dev deploy`, you can specify the steps from `build.yaml` that should be run. Unless there's a good reason to only deploy a particular service, you should use the full set listed below.
```

Comment:
```suggestion
After a change has been merged to the `main` branch, it can be deployed to the `default` namespace using the `prod_deploy` API endpoint. This will always use the current `HEAD`. Similar to a `dev deploy`, you can specify the steps from `build.yaml` that should be run. Unless there's a good reason to only deploy a particular service, you should use the set listed below. This is a partial list of steps that is specific to the CGP setup, and excludes services we don't use, for example the blog image fetcher.
```

Just adding a clarifying sentence based on your other comment in the original PR. Btw, I think the markdown linter might complain about line length here but I prefer soft-wraps to hard line breaks so this is fine to me.

Reply:
Yep, https://github.com/populationgenomics/team-docs/blob/main/.markdownlint.json#L3 should ignore line length.

Reply:
Thanks, added the suggestion!